docs/plugins/jenkins-pipeline-kubernetes.zh.md

```diff
@@ -18,4 +17,103 @@
+        wait: true
+        # the time to wait for any individual Kubernetes operation (like Jobs for hooks). This defaults to 5m0s
+        timeout: 5m
+        # whether to perform a CRD upgrade during installation
+        upgradeCRDs: true
+        # custom configuration. You can refer to [Jenkins values.yaml](https://github.com/jenkinsci/helm-charts/blob/main/charts/jenkins/values.yaml)
+        values_yaml: |
+          persistence:
+            # for prod env: the existent storageClass, please change it
+            # for test env: just ignore it, but don't remove it
+            storageClass: jenkins-pv
+          serviceAccount:
+            create: false
+            name: jenkins
+          controller:
+            serviceType: NodePort
+            nodePort: 32000
+            additionalPlugins:
+              # install "GitHub Pull Request Builder" plugin, see https://plugins.jenkins.io/ghprb/ for more details
+              - ghprb
+              # install "OWASP Markup Formatter" plugin, see https://plugins.jenkins.io/antisamy-markup-formatter/ for more details
+              - antisamy-markup-formatter
+            # Enable HTML parsing using OWASP Markup Formatter Plugin (antisamy-markup-formatter), useful with ghprb plugin.
+            enableRawHtmlMarkupFormatter: true
+            # Jenkins Configuraction as Code, refer to https://plugins.jenkins.io/configuration-as-code/ for more details
+            # notice: All configuration files that are discovered MUST be supplementary. They cannot overwrite each other's configuration values. This creates a conflict and raises a ConfiguratorException.
+            JCasC:
+              defaultConfig: true
+              # each key-value in configScripts will be added to the ${JENKINS_HOME}/casc_configs/ directory as a file.
+              configScripts:
+                # this will create a file named "safe_html.yaml" in the ${JENKINS_HOME}/casc_configs/ directory.
+                # it is used to configure the "Safe HTML" plugin.
+                # filename must meet RFC 1123, see https://tools.ietf.org/html/rfc1123 for more details
+  - name: jenkins-pipeline-kubernetes
+    # id of the tool instance
+    instanceID: default
+    # format: name.instanceID; If specified, dtm will make sure the dependency is applied first before handling this tool.
+    dependsOn: [ "jenkins.default" ]
+    # options for the plugin
+    options:
+      jenkins:
+        # jenkinsUrl, format: hostname:port
+        url: ${{jenkins.default.outputs.jenkinsURL}}
+        # jenkins user name, default: admin
+        user: admin
+        # jenkins password, you have 2 options to set the password:
+        # 1. use outputs of the `jenkins` plugin, see docs for more details
+        # 2. set the `JENKINS_PASSWORD` environment variable
+        # 3. fill in the password in this field(not recommended)
+        # if all set, devstream will read the password from the config file first.
```

Comment:
if all set, devstream will read the password from the config file or outputs from `jenkins` plugin first, then env var.